src/elements/__tests__/Banner.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { mount } from "enzyme"
+import React from "react"
+import { Banner } from "../Banner"
+
+describe("Button", () => {
+  it("displays the message", () => {
+    const message = "There was an error."
+    const wrapper = mount(<Banner message={message} />)
+    expect(wrapper.text()).toEqual(message)
+    expect(wrapper.find("CloseButton")).toHaveLength(0)
+  })
+
+  it("is dismissable", () => {
+    const message = "There was an error."
+    const wrapper = mount(<Banner dismissable message={message} />)
+    expect(wrapper.find("CloseButton")).toHaveLength(1)
+  })
+
+  it("disappears when dismissed", () => {
+    const message = "There was an error."
+    const wrapper = mount(<Banner dismissable message={message} />)
+    expect(wrapper.find("CloseButton")).toHaveLength(1)
+    wrapper.find("CloseButton").simulate("click")
+    expect(wrapper.find("CloseButton")).toHaveLength(0)
+  })
+})
```

Comment:
Thank you! 👍

Reply:
Yeah, I'm glad you said something - I didn't notice tests while poking around the first time because I'm so used to seeing them in the same folder as the implementation! When looking again, I noticed the `__tests_` folders and figuring out the precedent from looking through them. 👍

Reply:
haha! @jonallured - just went and looked and actually your message helped me catch something -- I realized I didnt move the tests when i created folders for stuff, will update today.